### PR TITLE
Fix min test case by doing mroe validation

### PR DESCRIPTION
**Pull request by carbotaniuman**

### CODE REVIEW

**Review by youknowone — APPROVED** [review]
Looks good as it is. How about `max`? If `min` and `max` has same rule for its arguments, can we have shared code for the argument process for the both function?

**Comment by carbotaniuman** [comment]
I refactored both functions into a single one that changes depending on arguments.

File: vm/src/function.rs [hunk 1]
```diff
@@ -196,12 +196,21 @@ impl FuncArgs {
                 T::arity().end(),
                 given_args,
             )))
-        } else if let Some(k) = self.kwargs.keys().next() {
-            Err(vm.new_type_error(format!("Unexpected keyword argument {}", k)))
         } else {
+            if let Some(err) = self.check_kwargs_empty(vm) {
+                return Err(err);
+            }
```
Comment (youknowone) [inline vm/src/function.rs:202]: is this moving related to `#![allow(clippy::suspicious_else_formatting)]`?

**Comment by carbotaniuman** [comment]
Previous test failure looked unrelated, retrying now

**Review by coolreader18 — APPROVED** [review]
Great work!

**Pull request merged**